tests/unit/framework/test_pipeline.py

```diff
@@ -106,13 +104,6 @@ def test_shut_down_calls_shut_down_on_pipeline_processors(self, _):
         for processor in processors:
             processor.shut_down.assert_called()
 
-    def test_setup_creates_connectors(self, _):
```

Comment:
Couldn't you refactor this test such that it also checks the properties `_input` and `_output`?

Reply:
not needed because it would test python itself...